scripts/install.sh

```diff
@@ -54,9 +54,13 @@ fi
 status "Downloading ollama..."
 curl --fail --show-error --location --progress-bar -o $TEMP_DIR/ollama "https://ollama.ai/download/ollama-linux-$ARCH"
 
-status "Installing ollama to /usr/bin..."
-$SUDO install -o0 -g0 -m755 -d /usr/bin
-$SUDO install -o0 -g0 -m755 $TEMP_DIR/ollama /usr/bin/ollama
+for BINDIR in /usr/local/bin /usr/bin /bin; do
+    echo $PATH | grep -q $BINDIR && break || continue
```

Comment:
maybe:
```
if [ -d $BINDIR ]; then
...
```
Just worried that this feels slightly obfuscated.

Reply:
The directory existing doesn't necessarily mean it's in the PATH. Most modern system have `/usr/local/bin` but it's almost never in the PATH